src/graphql/query/wwwHeader.graphql

```diff
@@ -19,6 +19,6 @@ query {
 		}
 	}
 	shop {
-		headerItemCount
+		nonTrivialItemCount
```

Comment:
This name changed, but I don't see it updated in any js?

Reply:
ahh so true